ChromProcess/Loading/peak_collection/peak_collection_from_csv.py

```diff
@@ -1,7 +1,7 @@
 from pathlib import Path
 from ChromProcess import Classes
 
-def peak_collection_from_csv(filename):
+def peak_collection_from_csv(filename,round_digits=3):
```

Comment:
added a keyword argument to allow for different rounding (preferential for peak_rt_from_csv), should not alter other function calls as the original value was 3 everywhere.

Reply:
nice, that's useful